src/main/java/appland/toolwindow/milestones/MilestoneActions.java

```diff
@@ -0,0 +1,52 @@
+package appland.toolwindow.milestones;
+
+import appland.milestones.UserMilestonesEditorProvider;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.NotNull;
+
+class MilestoneActions {
+    private static final Logger LOG = Logger.getInstance("#appland.milestones");
+
+    static void openQuickstart(@NotNull Project project) {
+        UserMilestonesEditorProvider.openUserMilestones(project);
+    }
+
+    /*static void installAppMapAgent(@NotNull Project project) {
```

Comment:
What is this code, @jansorg ?

Reply:
That code was part of the first milestone iteration before we decided to make a simplified version. I wanted to keep it, but we could also move it into a branch and remove it here.